www/src/config.ts

```diff
@@ -26,6 +26,7 @@ export type Frontmatter = {
 
 export const KNOWN_LANGUAGES = {
   en: "🇺🇸 English",
+  ar: "العربية 🇪🇬",
```

Comment:
AR instead of EG?

Reply:
EG is the Flag Emoji no the name of the country 👍🏼